fastapi/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8
```

Comment:
can you use the official tiagolo fastapi docker file here? 
`tiangolo/uvicorn-gunicorn-fastapi`

Reply:
Hey @espoirMur,

Sure, changed the docker image as requested to the pre-configured image of tiangolo. 

Cheers!